actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -180,7 +180,7 @@ workflows:
                     - run_sisyphus_check_indicies
             ### QUICK REPORT END ###
 
-            ### CHECK INDICIES START###
+            ### CHECK INDICES START###
```

Comment:
I'm sorry, I should've been more clear. You have this typo in many places in the workflow. :)